chapters/external-repository-identifiers.md

```diff
@@ -238,15 +238,41 @@ Examples:
 
 External Reference Site: [Software Heritage persistent IDentifiers](https://docs.softwareheritage.org/devel/swh-model/persistent-identifiers.html)
 
-### F.4.1 gitbom <a name="gitbom"></a>
+### F.4.2 gitoid <a name="gitoid"></a>
 
-Locator Format: gitoid":"\<git object type>":"\<hash algorithm>":"\<hash value>
-	
-Contextual Example: gitoid:blob:sha1:261eeb9e9f8b2b4b0d119366dda99c6fd7d35c64
+Locator Format: `"gitoid:"\<git object type>":"\<hash algorithm>":"\<hash value>`
+
+Locator Format Reference: <https://www.iana.org/assignments/uri-schemes/prov/gitoid>
+
+Current valid values are:
+* `<git object type>`: `blob`, `tree`, `commit`, `tag`. Only the `blob` type should be used for this identifier.
+* `<hash algorithm>`: `sha1`, `sha256`
+* `<hash value>` should be expressed as a hexadecimal string in lower case
+
+Contextual Examples:
+* `gitoid:blob:sha1:261eeb9e9f8b2b4b0d119366dda99c6fd7d35c64` is the git identifier of a software artifact using the SHA1 algorithm
+* `gitoid:blob:sha256:3557f7eb43c621c71483743d4b37059bb80933e7f71277c0c3b3846159d1f61c` is the git identifier of a software artifact using the SHA256 algorithm
+
+Documentation: `gitoid` stands for Git Object ID.  Git relies on a Merkle Tree to index stored objects. See <https://git-scm.com/book/en/v2/Git-Internals-Git-Objects>.
+
+### F.4.3 gitbom <a name="gitbom"></a>
+
+Locator Format: 
+`"gitbom:"\<git object type>":"\<hash algorithm>":"\<hash value>`
```

Comment:
Would it make sense to make this

`"gitbom:"\<gitoid uri>`

where `\<gitoid uri>` is defined above?